src/loki/polygon_search.cc

```diff
@@ -38,8 +38,6 @@ ring_bg_t PBFToRing(const valhalla::Options::Ring& ring_pbf) {
   for (const auto& coord : ring_pbf.coords()) {
     new_ring.push_back({coord.lng(), coord.lat()});
   }
-  // corrects geometry and handedness as expected by bg for rings
-  bg::correct(new_ring);
```

Comment:
`correct()` is not supporting `geographic` CS at lower versions it seems. but actually, let me look at this again in bit more detail.. it corrects a bunch of things, but the handedness check is probably most useful. we register a valhalla type as ring via a macro, where I guess the expected handedness is clockwise (the default for builtin ring type) and all tests are on clockwise rings as well. I'm not sure what'll happen if I add a counterclockwise test, so I'll do that real quick

Reply:
ok, seems to be fine.. I altered the avoids test to also test a counterclockwise ring and still passes